foundation/src/main/java/com/decathlon/vitamin/compose/foundation/Radius.kt

```diff
@@ -0,0 +1,28 @@
+package com.decathlon.vitamin.compose.foundation
+
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+val vtmnRadius = VitaminRadius(
```

Comment:
Instead of using directly this public property, we should create a local composition (like colors or typography) and use this local in all components (`VitaminTheme.shapes.radius100` for example).

Reply:
Note that the local shouldn't expose `dp` values but `RoundedCornerShape`.